### PR TITLE
post-process apple bundle

### PR DESCRIPTION
**Pull request by shakyShane**
https://app.asana.com/0/1201614831475344/1205448470130004/f

### CODE REVIEW

**Comment by shakyShane** [comment]
Current dependencies on/for this PR:
* main
  * **PR #725** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/725" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/725?utm_source=stack-comment).

File: packages/messaging/lib/webkit.js [hunk 1]
```diff
@@ -59,11 +59,6 @@ import { isResponseFor, isSubscriptionEventFor } from '../schema.js'
  * @implements {MessagingTransport}
  */
 export class WebkitMessagingTransport {
-    /** @type {WebkitMessagingConfig} */
-    config
```
Comment (jonathanKingston) [inline packages/messaging/lib/webkit.js:63]: Do we need these changes still?

Comment (shakyShane) [reply packages/messaging/lib/webkit.js:63]: These were always assigned in the constructor, so currently served no purpose.

Comment (jonathanKingston) [reply packages/messaging/lib/webkit.js:63]: I guess, just seems unrelated but it's fine.

File: packages/special-pages/package.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "@playwright/test": "^1.36.2",
     "http-server": "^14.1.1",
     "@duckduckgo/messaging": "*",
-    "esbuild": "^0.18.19",
```
Comment (jonathanKingston) [inline packages/special-pages/package.json:26]: Can this just inherit instead?

Comment (shakyShane) [reply packages/special-pages/package.json:26]: yeah I think we can move to top-level and then workspaces will make it work for special pages too :)

File: packages/messaging/lib/webkit.js (hunk 1 above)
Comment (shakyShane) [reply packages/messaging/lib/webkit.js:63]: it's not unrelated - these were the first errors I encountered (until I dug deeper), so when evaluating them and realising they were not needed, made sense to remove as part of this work :)

**Review by jonathanKingston — APPROVED** [review]

**Comment by shakyShane** [comment]
Thanks @jonathanKingston just going through last-min testing on Catalina - found something else problematic, but hopefully not much delay here

**Pull request merged**